### PR TITLE
Adding favicon

### PR DESCRIPTION
**Pull request by zaveryukha**

### CODE REVIEW

File: Packages/Sites/Sfi.Kateheo/Resources/Private/Partials/SharedResources.html [hunk 1]
```diff
@@ -17,22 +17,27 @@
...
+		<link rel="icon" type="image/png" href="{f:uri.resource(path: 'Legacy/icon/favicon-96x96.png', package: 'Sfi.Kateheo')}?{f:format.date(date:'now',format:'U')}" sizes="96x96">
+		<link rel="icon" type="image/png" href="{f:uri.resource(path: 'Legacy/icon/favicon-16x16.png', package: 'Sfi.Kateheo')}?{f:format.date(date:'now',format:'U')}" sizes="16x16">
+		<link rel="icon" type="image/png" href="{f:uri.resource(path: 'Legacy/icon/favicon-32x32.png', package: 'Sfi.Kateheo')}?{f:format.date(date:'now',format:'U')}" sizes="32x32">
+		<link rel="shortcut icon" href="{f:uri.resource(path: 'Legacy/icon/favicon.ico', package: 'Sfi.Kateheo')}?{f:format.date(date:'now',format:'U')}" type="image/x-icon">
 		<meta name="msapplication-TileColor" content="#ffffff">
-		<meta name="msapplication-TileImage" content="/mstile-144x144.png">
+		<meta name="msapplication-config" content="{f:uri.resource(path: 'Legacy/icon/ieconfig.xml', package: 'Sfi.Kateheo')}?{f:format.date(date:'now',format:'U')}" />
+		<link rel="manifest" href="{f:uri.resource(path: 'Legacy/icon//manifest.json', package: 'Sfi.Kateheo')}?{f:format.date(date:'now',format:'U')}">
```
Comment (dimaip) [inline Packages/Sites/Sfi.Kateheo/Resources/Private/Partials/SharedResources.html:38]: Double slash

**Comment by dimaip** [comment]
The large generated favicons have a poor quality. You can take this SVG http://sfi.ru/_Resources/Static/Packages/Sfi.Sfi/Images/SiteLogoRu.svg, enlarge it, then do print-screen and crop it, and get a quality PNG image, then feed it to http://realfavicongenerator.net/

**Comment by zaveryukha** [comment]
Ok. I'll update it. I found broken path to apple touch icons :( 
So a lot of work coming :)

**Comment by dimaip** [comment]
Just use http://realfavicongenerator.net/

**Pull request merged**